src/Microsoft.VisualStudio.VsixInstaller.Shared/Installer.cs

```diff
@@ -18,6 +18,11 @@ public static class Installer
         public static void Main(string[] args)
         {
             var rootSuffix = args[0];
+            if (rootSuffix == "NoExp")
```

Comment:
📝 I have no objection to supporting customized root suffix, but I would prefer to see it added as a property to `IdeFactAttribute`.

Reply:
I submitted #82 to provide a place (attribute) to specify the root suffix used for all test cases in an entire class.

Reply:
➡️ I submitted #85 to start addressing this 🎉